app/views/organisers/create.html.slim

```diff
@@ -0,0 +1 @@
+aa
```

Comment:
and we definitely need to delete this one ;) if you need please call `@organisers = User.where(role: 1)` in some else